src/spacegame3/gamedata/time/StarDateFormatter.java

```diff
@@ -5,14 +5,38 @@
 public abstract class StarDateFormatter {
 
         private static final String TIME_FORMAT_FOLDER_URL = "data/timeformat/";
+        private Path structurePath;
+        private String calenderName;
+        private String calenderDay;
+
```

Comment:
Those fields are completely unnecessary in the superclass, they aren't used in all the subclasses,
remove them